pages/1.10/release-notes/1.10.6/index.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.10.6
+title: Release Notes for 1.10.6
+menuWeight: 0
+excerpt:
+---
+
+These are the release notes for DC/OS 1.10.6.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.10.5/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+# Issues Fixed in DC/OS 1.10.6
+
+- DCOS-19865 - Fixed focus/change/blur behavior.
+- SOAK-68/DCOS-21451 - Fixed 404 error returned by adminrouter on Maration UI/Prevented the reuse of tcp sockets by AR's cache code [1.11].
+
+# Notable Changes in DC/OS 1.10.6
```

Comment:
Please add a note mentioning the new configuration option added here:
https://github.com/mesosphere/dcos-docs-site/pull/792

Reply:
@gpaul : I've added DCOS-16431. Can you review and approve it?